setup.py

```diff
@@ -14,6 +14,7 @@
         "pytest-timeout",
         "pytest-xdist",
         "build",
+        "toml",
```

Comment:
```suggestion
```

Why is a dead package listed here? `toml` was replaced by `tomli` (which is replaced by built-in `tomllib` in Python 3.11)?

Reply:
Please use the same structure we use elsewhere, which is tomli if <3.11 and tomllib (identical) otherwise.

Reply:
err, I guess we have to use `tomli_w` here instead?